### ISSUE TITLE
Improve localization extension installation message

### ISSUE THREAD

**Issue by joaomoreno** [Microsoft/vscode#62477]
![screenshot from 2018-11-02 15-12-26](https://user-images.githubusercontent.com/22350/47920777-34bb4300-deb3-11e8-84a7-87f24a695420.png)

Rephrase this to something like:

> In order to use VS Code in German, VS Code needs to restart.

Have a single button:

> Restart Now

We don't need a `No` button, since the notification can be dismissed.

**Comment by sandy081** [comment]
This message appears when a user installs a language and current message makes sense because it tells the user that there is a change in language is needed and not just restart. Saying just restart might be misleading the user. 


I would remove the `No` Button and leave the rest as is.
